src/main/java/fr/lunatech/timekeeper/services/responses/TimeSheetResponse.java

```diff
@@ -146,6 +153,9 @@ public final TimeSheetResponse filterToSixWeeksRange(Integer year, Integer month
                 .stream()
                 .filter(timeEntryResponse -> {
                     LocalDateTime startDateTime = timeEntryResponse.startDateTime;
+                    if(startDateTime == null){
```

Comment:
I don't understand the functional aspect of this... why we return true instead of false ?

Reply:
I agree that was a mistake, it should returns false to filter out the invalid entry.